data/page-worker/page-worker.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Page Worker</title>
+  </head>
+  <body>
+    <script>
+      window.addEventListener("addon-to-content", e => {
```

Comment:
Will use proper constants once this is converted to use a webpack build, but that won't happen until we do the html cacheing